File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.cs
using System;
using System.Collections.Generic;

public class Solution {
    public string solution(string s, string skip, int index) {
        
        List<char> alphabet = new List<char>();
        
        foreach (char c in "abcdefghijklmnopqrstuvwxyz")
        {
            if (!skip.Contains(c))
                alphabet.Add(c);
        }
        
        string answer = "";
        
        foreach (char c in s)
        {
            answer += alphabet[(alphabet.IndexOf(c) + index) % alphabet.Count];
        }
        
        return answer;
    }
}

File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alphabet = 'abcdefghijklmnopqrstuvwxyz'
    for i in skip:
        alphabet = alphabet.replace(i, '')    
        
    return ''.join(alphabet[(alphabet.index(i) + index) % len(alphabet)] for i in s)